checker/src/main/java/org/checkerframework/checker/nullness/qual/Nullable.java

```diff
@@ -29,5 +30,6 @@
 @DefaultInUncheckedCodeFor({TypeUseLocation.RETURN, TypeUseLocation.UPPER_BOUND})
 @Documented
 @Retention(RetentionPolicy.RUNTIME)
+@DefaultFor(TypeUseLocation.TYPE_DECLARATION)
```

Comment:
I think improving the documentation for `TypeUseLocation.TYPE_DECLARATION` would be important.
Using this default is different from other defaults - the behavior between a class explicitly annotated as `@Nullable` is different from a class that gets this default `@Nullable`.

Reply:
I'll update [this section](https://checkerframework.org/manual/#annotations-on-class-declarations) of the manual.
Do you this I should add documentation here and/or at the declaration of TypeUseLocation, too?